### DISCUSSION TITLE
query dark mode

### DISCUSSION THREAD

**Discussion by hros** [fonsp/Pluto.jl#1971] [Questions about using Pluto] [answered]
Is there a way to query, from within a notebook cell, whether the notebook is displayed in light or dark mode?
When the notebook displays graphics, specifically a circuit diagram with lines whose color is hard coded, I would like to change the line color based on the display mode of the notebook

**Comment by fonsp** [comment, marked as answer]
Hey! You can only do this from within CSS or JS, using https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme (We don't support a way to get this from Julia because there are situations where the same output will be displayed in multiple browsers/settings.)
